python/sglang/srt/managers/schedule_batch.py

```diff
@@ -1133,6 +1138,9 @@ def get_model_worker_batch(self):
             input_embeds=self.input_embeds,
         )
 
+    def get_hf_eos_token_id(self) -> Optional[Set[int]]:
```

Comment:
This is still redundant. Can we cache this value `class Scheduler:` and pass it to `Req` here https://github.com/sgl-project/sglang/blob/dc3bee4815183d19761082b9ac90ada72d0151ef/python/sglang/srt/managers/scheduler.py#L497

Reply:
Done